example/trino_standalone/main.tf

```diff
@@ -65,10 +66,24 @@ module "trino" {
     vault_kv_access_key_name = ""
     vault_kv_secret_key_name = ""
   }
+  postgres_service = {
+    service_name  = module.postgres.service_name
+    port          = module.postgres.port
+    username      = module.postgres.username
+    password      = module.postgres.password
+    database_name = module.postgres.database_name
+  }
+  postgres_vault_secret = {
+    use_vault_provider      = false
+    vault_kv_policy_name    = ""
+    vault_kv_path           = ""
+    vault_kv_field_username = ""
+    vault_kv_field_password = ""
+  }
```

Comment:
Just to clarify, in the current example do you (@claesgill ) use the same `postgres` instance for both `hive` and `trino`?

As I understood, you connect to the same `postgres` instance, to the same database that `hive` use and you use the same credentials that `hive` use. Am I right?

Reply:
Yes, that's right. Should I add another Postgres? Or WDYT?

Reply:
I think at first iteration, I would be happy if you add this info to readme.md of example.

Reply:
Done: Updated the Readme example.